bitnami/discourse/README.md

```diff
@@ -48,207 +48,193 @@ The command removes all the Kubernetes components associated with the chart and
-| `persistence.selector`       | Selector to match an existing Persistent Volume (this value is evaluated as a template)   | `{}`                                                    |
-| `updateStrategy`             | Update strategy of deployment                                                             | `{type: "RollingUpdate"}`                               |
-| `podAnnotations`             | Additional pod annotations                                                                | `{}`                                                    |
-| `podLabels`                  | Additional pod labels                                                                     | `{}` (evaluated as a template)                          |
-| `commonLabels`               | Labels to be added to all deployed resources                                              | `{}` (evaluated as a template)                          |
-| `commonAnnotations`          | Annotations to be added to all deployed resources                                         | `{}` (evaluated as a template)                          |
-| `podAntiAffinityPreset`      | Pod anti-affinity preset. Ignored if `affinity` is set. Allowed values: `soft` or `hard`  | `soft`                                                  |
-| `nodeAffinityPreset.type`    | Node affinity preset type. Ignored if `affinity` is set. Allowed values: `soft` or `hard` | `""`                                                    |
-| `nodeAffinityPreset.key`     | Node label key to match Ignored if `affinity` is set.                                     | `""`                                                    |
-| `nodeAffinityPreset.values`  | Node label values to match. Ignored if `affinity` is set.                                 | `[]`                                                    |
-| `affinity`                   | Affinity for pod assignment                                                               | `{}` (evaluated as a template)                          |
-| `nodeSelector`               | Node labels for pod assignment.                                                           | `{}` (evaluated as a template)                          |
-| `tolerations`                | Tolerations for pod assignment.                                                           | `[]` (evaluated as a template)                          |
+| Name                         | Description                                                                               | Value                |
+| ---------------------------- | ----------------------------------------------------------------------------------------- | -------------------- |
+| `kubeVersion`                | Force target Kubernetes version (using Helm capabilites if not set)                       | `nil`                |
```

Comment:
This parameter (along with `nameOverride`, `fullnameOverride`, `commonLabels` and `commonAnnotations`) should go into this "common parameters" section. 

The rest of parameters should be moved to the "Discourse parameters" section. 

See https://github.com/bitnami/charts/tree/master/bitnami/jenkins#common-parameters

Reply:
Done in a1c39ee. Thanks!